js/common/enums.js

```diff
@@ -0,0 +1,12 @@
+const keyboardKeys = {
+  ESC: ('Escape' || 'Esc'),
```

Comment:
```suggestion
  ESC: 'Escape',
```
Этого будет достаточно